### ISSUE TITLE
Erroneous rendering

### ISSUE THREAD

**Issue by nkh** [Titor8115/mandown#38]
This markdown :

1   # NAME
  1
  2 ftl - terminal file manager, tmux based
  3
  4 ## Commands shortcuts (⇑: alt-gr, ⇈: shift+alt-gr)
  5 ```
  6 q Q @  ) close, > &3, keep shell         ?       ) show help
  7 a      ) kill sound preview              j B     ) next entry
  8 b ⇑b   ) find/fzf entry in directory     k A     ) previous entry
  9 c      ) copy selection                  h D     ) parent directory
 10 ⇑c     ) compress/decompress             l C ''  ) cd or open file
 11 d      ) delete                          5 6     ) page down/up
 12 ...
 13
 14 for a complete list see resources/.config/ftl/help.md or press '?' in ftl
 15
 16
 17 ```
 18
 19 ![Screenshot](https://raw.github.com/nkh/ftl/master/screenshots/ftl.png)
 20 ![Screenshot](https://raw.github.com/nkh/ftl/master/screenshots/image_preview.png)
 21
 22 # DOCUMENTATION
 23
 24 There are many promising file managers for the terminal, from fff to nnn, clifm, ranger, vimfm, broot, etc ...
 

renders as:

README.md(7)

NAME
       NAME
       ftl - terminal file manager, tmux based

Commands shortcuts (⇑: alt-gr, ⇈: shift+alt-gr)
       ``` q Q @ ) close, > &3, keep shell ? ) show help a ) kill sound preview j B ) next entry b ⇑b ) find/fzf entry in directory k A ) previous entry c ) copy selection h D ) parent
       directory ⇑c ) compress/decompress l C '' ) cd or open file d ) delete 5 6 ) page down/up ...

       for a complete list see resources/.config/ftl/help.md or press '?' in ftl

       ```

       Screenshot Screenshot

NAME
       DOCUMENTATION
       There are many promising file managers for the terminal, from fff to nnn, clifm, ranger, vimfm, broot, etc ...

the errors:
# NAME
->    NAME is written twice
    
```
 q Q @  ) close, > &3, keep shell         ?       ) show help
 a      ) kill sound preview              j B     ) next entry

-> rendered all on one line

![Screenshot](https://raw.github.com/nkh/ftl/master/screenshots/ftl.png)
-> url not shown

# DOCUMENTATION
 ->  NAME is rendered again!, DOCUMENTATION is not indented properly

**Comment by Titor8115** [comment]
Yeah `# h1` is treated as the `NAME` section of man-page, But since markdown allows paragraphs under h1, h1 itself is used as part of the sectioning

Just do: (pre tag is for GitHub formatting)

<pre>
# ft
terminal file manager, tmux based
</pre>

It will render as:
<pre>
NAME: ft
       terminal file manager, tmux based
</pre>

**Comment by Titor8115** [comment]
Yeah `# h1` is treated as the `NAME` section of man-page, But since markdown allows paragraphs under h1, h1 itself is used as part of the sectioning

Just do: (pre tag is for GitHub formatting)

<pre>
# ft
terminal file manager, tmux based
</pre>

It will render as:
<pre>
NAME: ft
       terminal file manager, tmux based
</pre>
